---
string tag1
float64 pos_x1
float64 pos_y1
float64 pos_z1
float64 or_x1
float64 or_y1
float64 or_z1
float64 or_w1
string tag2
float64 pos_x2
float64 pos_y2
float64 pos_z2
float64 or_x2
float64 or_y2
float64 or_z2
float64 or_w2
string tag3
float64 pos_x3
float64 pos_y3
float64 pos_z3
float64 or_x3
float64 or_y3
float64 or_z3
float64 or_w3
string tag4
float64 pos_x4
float64 pos_y4
float64 pos_z4
float64 or_x4
float64 or_y4
float64 or_z4
float64 or_w4